snippets/worker-latency-profile-intro.adoc

```diff
@@ -8,16 +8,20 @@
 
 :_content-type: SNIPPET
 
-All nodes send heartbeats to the Kubernetes Controller Manager Operator (kube controller) in the {product-title} cluster every 10 seconds, by default. If the cluster does not receive heartbeats from a node, {product-title} responds using several default mechanisms.
+All nodes send heartbeats to the Kubernetes Controller Manager (kube controller) in the {product-title} cluster every 10 seconds, by default. If the cluster does not receive heartbeats from a node, {product-title} responds using several default mechanisms.
 
-For example, if the Kubernetes Controller Manager Operator loses contact with a node after a configured period:
+For example, if the Kubernetes Controller Manager loses contact with a node after a configured period, the default behavior is:
 
 . The node controller on the control plane updates the node health to `Unhealthy` and marks the node `Ready` condition as `Unknown`.
 
 . In response, the scheduler stops scheduling pods to that node.
 
-. The on-premise node controller adds a `node.kubernetes.io/unreachable` taint with a `NoExecute` effect to the node and schedules any pods on the node for eviction after five minutes, by default.
+. The Kubernetes Controller Manager adds a `node.kubernetes.io/unreachable` taint with a `NoExecute` effect to the node and schedules any pods on the node for eviction after five minutes, by default.
```

Comment:
Node life cycle controller taints the nodes, https://github.com/kubernetes/kubernetes/blob/bd0ac4cba33f40195b4ffab5463f2b5397216400/pkg/controller/nodelifecycle/node_lifecycle_controller.go#L69

Reply:
Thank you, updated!